Testnet release plan.md

```diff
@@ -0,0 +1,55 @@
+# Testnet versions
```

Comment:
It feels like some of the items should be pushed out to post-prod. Mobile wallet etc are fine shortcomings in early stages, but roll-up contracts etc are not. Need to ensure we are code complete so the audit can start ASAP. Is this really a bare bone MVP?

Reply:
fair enough. 
I felt like that would be something essential to explore, but you're right, could be tackled post prod.

Also, bear in mind that this list is not taking us to prod.

Reply:
sorry, to answer your question. This is not a bear-bone MVP. To get there we'll probably need 1 or 2 more milestones, but we'll discover the exact items, once we get closer